dask/dataframe/io/hdf.py

```diff
@@ -4,27 +4,31 @@
 from glob import glob
 import os
 from threading import Lock
-import multiprocessing
+import multiprocessing as mp
 import uuid
 from warnings import warn
 
 import pandas as pd
-import dask
 from toolz import merge
 
 from ...async import get_sync
 from ...base import tokenize
 from ...compatibility import PY3
 from ...context import _globals
 from ...delayed import Delayed, delayed
-import dask.multiprocessing
+from ...import multiprocessing
 
 from ..core import DataFrame, new_dd_object
 
-from ...utils import build_name_function
+from ...utils import build_name_function, SerializableLock
 
 from .io import _link
 
+try:
+    import distributed
+except ImportError:
+    distributed = None
```

Comment:
This would add a non-trivial amount to the import time

```python
In [1]: import dask

In [2]: %time import distributed
CPU times: user 252 ms, sys: 244 ms, total: 496 ms
Wall time: 271 ms
```

Perhaps we should do the import within the function?